src/components/POLine/POLineForm.js

```diff
@@ -89,7 +94,7 @@ import styles from './POLineForm.css';
 import { createPOLDataFromInstance } from './Item/util';
 import { useManageDonorOrganizationIds } from './hooks';
 
-const GAME_CHANGER_FIELDS = ['isPackage', 'orderFormat', 'checkinItems', 'packagePoLineId', 'instanceId'];
+const GAME_CHANGER_FIELDS = ['isPackage', 'orderFormat', 'checkinItems', 'packagePoLineId', 'instanceId', 'customFields'];
```

Comment:
"Game changer" fields are basically fields, which influence other fields in the form. Do the `customFields` field is one of them?

Reply:
@usavkov-epam  Thanks for your response!
I need to add `customFields` to `GAME_CHANGER_FIELDS` to transfer the customFields values defined in the template.
Is there another way to do that?

Reply:
At first glance, it looks like the values should be taken from the template without changing the game-changer array. For all registered form fields (when the change game fields played a role), a check is carried out for the presence of a value in the associated template; if there is one, it is substituted into the form.

ref https://github.com/folio-org/ui-orders/blob/master/src/components/POLine/POLineForm.js#L199

If you leave the `GAME_CHANGER_FIELDS` untouched, do you have problems with the population of custom fields from the template?

Reply:
Exactly, if i leave `GAME_CHANGER_FIELDS` untouched, the `customFields` values from the templates are not taken over to the form.

Reply:
Hmm, strange, it works for me locally.
But if your approach works it's acceptable I think

Reply:
Well, it works for `POForm` but not for `POLineForm`

Reply:
The problem here is the asynchronous nature of `<EditCustomFieldsRecords>`, which renders its `<Field>` components only after successfully fetching data from the custom-fields API. The `populateFieldsFromTemplate()` method may be invoked before these `<Field>` components have registered with the form, leading to inconsistent behavior. Even if `"customFields"` is added to `GAME_CHANGER_FIELDS`, it does not guarantee that the fields are present when attempting to populate them, resulting in a race condition.

Reply:
Thanks @alb3rtino for the hint! 
`customFields` is no longer a `GAME_CHANGER_FIELDS`, I removed it.
We have to add a callback function to `<EditCustomFieldsRecords>` so we know when the values are loaded.